pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -202,6 +209,18 @@ private void setupHelixSystemProperties() {
             CommonConstants.Helix.DEFAULT_FLAPPING_TIME_WINDOW_MS));
   }
 
+  private void setupHelixClusterConstraints() {
+    String maxMessageLimit = _config.getString(CommonConstants.Helix.CONFIG_OF_HELIX_INSTANCE_MAX_MESSAGES,
+        CommonConstants.Helix.DEFAULT_HELIX_INSTANCE_MAX_MESSAGES);
+    Map<ClusterConstraints.ConstraintAttribute, String> constraintAttributes = Maps.newHashMap();
```

Comment:
No need to use guava for this. We have experienced several backward-incompatible issues with guava.
```suggestion
    Map<ClusterConstraints.ConstraintAttribute, String> constraintAttributes = new HashMap()<>;
```

Reply:
Good to know this. thanks